nuxeo-coldstorage-web/src/main/resources/web/nuxeo.war/ui/coldstorage/nuxeo-retrieve-content-from-coldstorage-button.html

```diff
@@ -29,7 +29,7 @@
       </template>
     </dom-if>
 
-    <nuxeo-dialog id="contentToMoveDialog" with-backdrop>
+    <nuxeo-dialog id="contentToRetrieveDialog" with-backdrop>
```

Comment:
Thanks a lot for catching this copy/paste miss. I just checked on the other branches, and we're calling it contentFromRetrieveDialog. Could we have the same name here for consistency? Thanks! :)

Reply:
Yes, I noticed that when I did the forward port. I'll change here